ipympl/backend_nbagg.py

```diff
@@ -540,7 +540,8 @@ def show(block=None):
 
 
 def flush_figures():
-    if rcParams['backend'] == 'module://ipympl.backend_nbagg':
+    backend = matplotlib.get_backend()
```

Comment:
This is considered the correct way to determine the current Matplotlib backend, it just calls `rcParams['backend']` internally.